pkg/distributor/distributor.go

```diff
@@ -437,6 +444,8 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 		}),
 
 		PushMetrics: newPushMetrics(reg),
+		now:         defaultNow,
```

Comment:
Style comment so feel free to ignore, instead of defining default functions above you could just set them here. e.g. 
```suggestion
		sleep:       time.Sleep,
		now:         time.Now,
```